CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 The following changes have been implemented but not released yet:
 
+### Bugs fixed
+
+- When writing non-RDF data, the request headers were incorrectly set, resulting in incorrect headers.
```

Comment:
This seems a bit redundant :P

```suggestion
- When writing non-RDF data, the request headers were incorrectly set.
```

or posisbly

```suggestion
- When writing non-RDF data, the request headers were incorrectly set, resulting in the save failing (when?).
```

Reply:
The headers were really incorrectly incorrect.